### ISSUE TITLE
Builder.parse does not parse the xml from view

### ISSUE THREAD

**Issue by kefahB** [NativeScript/NativeScript#9015]
Hi @davecoffin,

I work on [this PR](https://github.com/dapriett/nativescript-google-maps-sdk/pull/440) in order to update `nativescript-google-maps-sdk` to `NS7`.

I've noticed that the `Builder.parse` does not parse the xml as expected .. on `NS 6.x` the plugin work just fine with same Builder.parse and with NS7 it does not work at all [please see here](https://github.com/kefahB/nativescript-google-maps-sdk/blob/597813945083bb6114b8ad8abe615c89779536ea/src/map-view-common.ts#L154)

to reproduce : 

This is `maps:mapView.infoWindowTemplate` 
```xml
<StackLayout orientation="vertical" width="200" height="150" >
                    <Label text="{{title}}" className="title" width="125"   />
                    <Label text="{{snippet}}" className="snippet" width="125"   />
                    <Label text="{{'LAT: ' + position.latitude}}" className="infoWindowCoordinates"  />
                    <Label text="{{'LON: ' + position.longitude}}" className="infoWindowCoordinates"  />
</StackLayout>
```

```ts
// doing this doesn't work
return Builder.parse(this.infoWindowTemplate, this); // return StackLayout(4)@main-page.xml:20:17;

// this work but is not a costume infoWindowTemplate any more
Builder.parse(`<StackLayout orientation="vertical" width="200" height="150" >
                <Label text="{{title}}" className="title" width="125"   />
                <Label text="{{snippet}}" className="snippet" width="125"   />
                <Label text="{{'LAT: ' + position.latitude}}" className="infoWindowCoordinates"  />
                <Label text="{{'LON: ' + position.longitude}}" className="infoWindowCoordinates"  />
            </StackLayout>`);

```

Any suggestion please ?

Thanks

**Comment by NathanWalker** [comment]
@kefahB hi - I noticed the PR was merged there and was trying to follow some of the comments; Did you all end up finding a solution on the Builder.parse issue mentioned or is this still pending?

**Comment by kefahB** [comment]
Hi @NathanWalker,

I’ve found a way to get the plugin work but the builder still doesn’t work!

There is change with builder between NS6 and 7 ? 

I have another question please, there is a way to implement 2 delegates on same class with TS ?

Thank you

**Comment by NathanWalker** [comment]
Ok good to know will try to check that out this week (re: builder).

As far as 2 delegates, if I understand you correctly, yes.
```
@NativeClass()
class CustomClass extends NSObject implements DelegateA, DelegateB {
  static ObjCProtocols = [DelegateA, DelegateB];
 
  // implement both delegate methods...
}
```
That what you mean?

**Comment by kefahB** [comment]
Waiting for news (builder)

Yes exactly, I’ve already tried to implementing this on the MpView but one delegate prime but not both !

**Comment by NathanWalker** [comment]
@kefahB depends on how the owner receiver expects things. native object's often contain a singular delegate setter which expects a certain protocol. In your case I would guess there's 2 delegate setters for the native class? In that case better to just use 2 distinct TS configured custom delegate classes.

**Comment by kefahB** [comment]
@NathanWalker you are right, the owner receiver expect `GMSMapViewDelegate` and I try to implement `CLLocationManagerDelegate` 

I've asked [here](https://stackoverflow.com/questions/64554261/how-can-implements-2-delegate-with-typescript) with more details, I tried many concept but always one delegate are handled!

**Comment by kefahB** [comment]
@NathanWalker I found the issue on the mapview delegate ! it was very simple the `didTapMyLocationButtonForMapView` should return false instead of true.

**Comment by NathanWalker** [comment]
ok awesome thanks for confirming!
